utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -929,12 +934,13 @@ function Set-DeploymentExamplesSection {
         ''
     )
 
-    # Get resource type and make first letter upper case. Requires manual handling as ToTitleCase lowercases everything but the first letter
-    $providerNamespace = ($fullModuleIdentifier.Split('/')[0] -split '\.' | ForEach-Object { $_.Substring(0, 1).ToUpper() + $_.Substring(1) }) -join '.'
-    $resourceType = $fullModuleIdentifier.Split('/')[1]
-    $resourceTypeUpper = $resourceType.Substring(0, 1).ToUpper() + $resourceType.Substring(1)
-
-    $FullModuleIdentifier = "$providerNamespace/$resourceType"
+    # Get moduleName as $fullModuleIdentifier leaf
+    $moduleName = $fullModuleIdentifier.Split('/')[1]
+    # Convert moduleName from kebab-case to camelCase
+    $First, $Rest = $moduleName -Split '-',2
+    $moduleNameCamelCase = $First.Tolower() + (Get-Culture).TextInfo.ToTitleCase($Rest) -Replace '-'
+    # Convert moduleName from kebab-case to PascalCase
+    $moduleNamePascalCase = (Get-Culture).TextInfo.ToTitleCase($moduleName) -Replace '-'
```

Comment:
Do you have an example of the expected input & output?

When I run this with e.g. 'Microsoft.Storage/storageAccounts' I get
- $moduleNameCarmelCase = 'storageaccounts'
- $moduleNamePascalCase = 'Storageaccounts'

Wouldn't we want something like 
- $moduleNameCarmelCase = 'storageAccounts'
- $moduleNamePascalCase = 'StorageAccounts'
?

Reply:
Module names are now kebab-case so that is the intended input

Example:
$fullModuleIdentifier = 'storage/storage-accounts'
$moduleName = 'storage-accounts'
$moduleNameCarmelCase = 'storageAccounts'
$moduleNamePascalCase = 'StorageAccounts'

Reply:
ah it is the other way around. My bad. 
Thanks for clarifying.